active/CornerRadius/CornerRadiusToThickness2.md

```diff
@@ -0,0 +1,149 @@
+
+# Background
+
+_This spec updates the existing [CornerRadiusToThicknessConverter](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.Primitives.CornerRadiusToThicknessConverter) class with new filters._

```

Comment:
Curiously, the existing documentation lacks these illustrations and examples.

Reply:
Recommend: Carry over to the doc update